pipelines/azure-pipelines.load-test.yml

```diff
@@ -106,10 +124,9 @@ steps:
   condition: succeededOrFailed()
   displayName: 'RESULTS: Publish Load Test Artifacts'
 
-- script: terraform destroy -auto-approve
+- script: terraform destroy -var="JMETER_ENVIRONMENT_FILE=${{ parameters.testenvironmentfile }}" -var="JMETER_WORKERS_COUNT=${{ parameters.workercount }}" -var="JMETER_LOAD_CONFIG_FILE=${{ parameters.loadconfigurationfile }}" -auto-approve
```

Comment:
@hepsi204 
Should this be `terraform destroy -auto-approve` ? Or is it some partial update?

Reply:
yup it shouldd be. will be corrected. thanks

Reply:
corrected